core/client/fs/src/main/java/alluxio/client/file/cache/DefaultMetaStore.java

```diff
@@ -67,8 +65,7 @@ public void addPage(PageId pageId, PageInfo pageInfo) {
     mPageMap.put(pageId, pageInfo);
     mBytes.addAndGet(pageInfo.getPageSize());
     Metrics.SPACE_USED.inc(pageInfo.getPageSize());
-    mPages.incrementAndGet();
-    Metrics.PAGES.inc();
+    Metrics.PAGES.inc(mPageMap.size() - Metrics.PAGES.getCount());
```

Comment:
is this should be mPageMap.size（）？

Reply:
Not really, we cannot increase it by the mPageMap.size.  because of the interface of the metrics counter, I cannot assign a value to it directly, so I made a diff between mPageMap.size and the current value of the metrics.

Reply:
Nice! Thank you @maobaolong ! could you help take another look on this pr? I also added some unit test to cover this logic. Thx!